platformio.ini

```diff
@@ -14,7 +15,7 @@ lib_deps =
 	lincomatic/LiquidTWI2@^1.2.7
 	olikraus/U8g2@^2.28.8
 
-[env]
+[common_embedded]
```

Comment:
embedded env should have:
`test_ignore = test_desktop` so it does not execute tests meant only for the host machine